### DISCUSSION TITLE
I have been waiting for more than three months, almost no updates, just relying on waiting for updates. Why? Looking forward to a stable version

### DISCUSSION THREAD

**Discussion by DreamPWJ** [arthelokyo/astrowind#182] [Q&A]
I have been waiting for a stable version for project development. As an open source project, I know I have no right to criticize it. I just hope for a better stable version, better configuration options, support for low-end browsers, etc

**Comment by DreamPWJ** [comment]
Does Tinacms support integration? The biggest challenge I am facing now is dynamically adding news information. Currently, all blogs are in MD format, which is not friendly for non-technical personnel. Can we support editor mode without the need for backend and database? Ordinary personnel can also dynamically add and edit news without the need for code changes  @prototypa

**Comment by jjaimealeman** [comment]
This template/theme is more than STABLE for a production site, as I have used it for various public client facing sites and I have faced `0` issues with it.

> support for low-end browsers 

Astro by its very nature _(being a web framework)_ is geared toward _"modern browsers"_, as are mostly all modern day frameworks. If you require support for older browsers, then perhaps Astro is not for you.

Have you checked out 11ty or Hugo?
